### ISSUE TITLE
AttributeError

### ISSUE THREAD

**Issue by evasehr** [pycaret/pycaret#16]
Hi - I really enjoy the package! However, when executing the script on a server, I get the following error:

Traceback (most recent call last):
  File "RF_pycaret.py", line 15, in <module>
    model1 = create_model('rf')
  File "/home/../.conda/envs/pycaret/lib/python3.8/site-packages/pycaret/regression.py", line 1691, in create_model
    display_id = display_.display_id
AttributeError: 'NoneType' object has no attribute 'display_id'

Any clue what's going on?
Thanks for your help!

**Comment by pycaret** [comment]
Hi,

Can you try it on Jupyter notebook. 

What IDE are you using? I will try to reproduce the issue at my end.

Thanks.

**Comment by evasehr** [comment]
hi - thanks for your quick reply. locally (with jupyter lab) it seems to work, at least until the tuning step, which is running right now. on the server, well, I didn't use any IDE, but vim...

**Comment by madazone** [comment]
I got the same issue after running the compare_models
    models = compare_models(sort='Precision')

  File "/Users/madazone/Workspace/blincks/env3.7/lib/python3.7/site-packages/pycaret/classification.py", line 3185, in compare_models
    display_id = display_.display_id
AttributeError: 'NoneType' object has no attribute 'display_id'

**Comment by pycaret** [comment]
Hi,

To avoid this please pass verbose = False in function. PyCaret uses html to display progress which I believe is not supported in all environment. Hence for unattended runs on servers verbose must be set to False.

Thanks.

**Comment by madazone** [comment]
compare_models does not support the verbose mode

**Comment by pycaret** [comment]
@madazone Are you using notebook?

**Comment by madazone** [comment]
No. I am running it directly from python3.7

**Comment by xhlubuntu** [comment]
i encounter the same problem. i'm running directly from python too.

**Comment by tech-a-go-go** [comment]
@pycaret 
I specified the verbose arg like this `compare_models(verbose = False)` but getting the following error.
```
TypeError: compare_models() got an unexpected keyword argument 'verbose'
```

My tooltip in VSCODE says 
```
def compare_models(blacklist=None, fold=10, round=4, sort='Accuracy', turbo=True)
...
blah blah blah
```

Cheers

**Comment by pycaret** [comment]
@madazone 

### Feature now available in pycaret-nightly.

You can install **pycaret-nightly** using "pip install pycaret-nightly".

New parameter **html** is now added in setup. By default it is set to _True_. If you are using PyCaret outside of notebook environment, please pass html = False in **setup**.

Can you please test this and close the issue if the enhancement addresses the original issue.

It will be shipped to PyCaret 2.0x in few weeks.

**Comment by sACPDX** [comment]
Very impressive library.  I am trying to use PyCaret with Python 3.8 and Streamlit.

I am experiencing a similar issue running this in python and streamlit locally.  I installed the pycaret-nightly as recommended.

VS Code Terminal shows results of Setup() as running fine when setup runs in Streamlit.  
Setup Succesfully Completed.
                      Description      Value
0                      session_id        123
1               Transform Target       False
2         Transform Target Method       None ...

However, Python app with Streamlit does not consider the setup to be complete.  Note that it did before I installed pycaret-nightly.

Here is the code:
exp_reg101 = setup(data = X_original, target = 'Quantity', session_id=123, silent=True, html=False)
if exp_reg101:
   st.success("PyCaret setup complete") 
   st.write("Results", compare_models())
else:
   st.warning("PyCaret setup did not complete")

welcome any suggestions.
